gradle.properties

```diff
@@ -1,20 +1,23 @@
-# Project-wide Gradle settings.
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
+GROUP=app.futured.donut
+VERSION_NAME=2.2.0-SNAPSHOT
```

Comment:
I understand that we need to update this property before every release? We should create some sort of publishing guide, this might get easily forgotten.

Reply:
Yes. Agree with the guide.